src/modifiers.rs

```diff
@@ -63,7 +64,7 @@ use response::{WriteBody, BodyReader};
 impl Modifier<Response> for Mime {
     #[inline]
     fn modify(self, res: &mut Response) {
-        res.headers.set(headers::ContentType(self))
+        res.headers.insert(headers::CONTENT_TYPE, self.as_ref().parse().unwrap()); // TODO: error handling?
```

Comment:
What do we do with these errors?

Reply:
I think in this case, there is no error handling necessary, as a Mime should always parse into a valid string for HeaderValue.